cg-spec/editors_draft.html

```diff
@@ -3068,8 +3071,39 @@ <h2>Referential opacity</h2>
         -->
         </pre>
 
-        <p>Note that the current semantics does not forbid the additional inferences to be drawn. However, they would require additional knowledge and an appropriate <a data-cite="RDF11-MT#dfn-semantic-extension">semantic extension</a> (such as the one discussed in <a href="#selective-ref-transparency"></a>). This makes RDF-star less readily usable for (but not incompatible with) that latter kind of use-cases.</p>
+      </section>
+
+      <section id="selective-ref-transparency">
+        <h2>Selective referential transparency</h2>
+
+        <p>Referencial opacity is well suited for use-cases where the syntactical form of the annotated statements may be significant (like <a href="../UCR/rdf-star-ucr.html#annotate-statements-from-other-named-graphs-and-datasets">annotating statements from other graphs</a> or <a href="../UCR/rdf-star-ucr.html#annotate-commit-deltas">annotating commit deltas</a>). On the other hand, other use-cases would expect some degree of referential transparency. Consider the case of <a href="../UCR/rdf-star-ucr.html#meta-properties-over-properties">attributed/evidenced triples</a>. The annotation is understood to be about the fact asserted by the triple rather than the triple itself. In this use case, one might then expect more inferences than provided by the base semantics. Consider the following example.</p>
+
+        <pre id="selective-ref-opacity"
+        data-transform="updateExample"
+        data-content-type="text/turtle"
+        class="nohighlight example"
+        >
+        <!--
+        << dbr:Linköping dbo:populationTotal "104232"^^xsd::nonNegativeInteger >>
+          :on "2010-12-31"^^xsd:date ;
+          :source <https://dbpedia.org/data/Linköping.ttl> .
+
+        ### Candidate entailment #1
+
+        << dbr:Linköping dbo:populationTotal "0104232"^^xsd::nonNegativeInteger >>
+          :on "2010-12-31"^^xsd:date .
+
+        ### Candidate entailment #2
+
+        << dbr:Linköping dbo:populationTotal "0104232"^^xsd::nonNegativeInteger >>
+          :source <https://dbpedia.org/data/Linköping.ttl> .
+        -->
+        </pre>
+
+        <p>One could argue that the candidate entailment #1 is desirable because the property `:on` is understood to be about the <em>statement</em> made by the subject triple (that statement was true on the given date, regardless of its RDF expression). On the other hand, the candidate entailment #2 may be considered not desirable because the predicate `:source` is about the triple itself (that specific triple was parsed from the given Turtle file).</p>
```

Comment:
```suggestion
        <p>One could argue that the candidate entailment #1 is desirable because the property `:on` is understood to be about the <em>statement</em> made by the subject triple (that statement was true on the given date, regardless of its RDF expression, in this case using an object value equivalent to `104232`). On the other hand, the candidate entailment #2 may be considered not desirable because the predicate `:source` is about the triple itself (that specific triple was parsed from the given Turtle file).</p>
```

Reply:
works for me. thx